packages/xrpl/snippets/src/multisigning.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  multisign,
+  Client,
+  AccountSet,
+  convertStringToHex,
+  SignerListSet,
+} from '../../dist/npm'
```

Comment:
Oh wait - should this import from `xrpl` instead of `dist/npm`?

Reply:
(Can check what the other snippets do)

Reply:
Others also import from that same `dist/npm` repo so I dont think it's an issue :)